### ISSUE TITLE
Feature request: sigmoid postprocessor

### ISSUE THREAD

**Issue by krishna-esrlabs** [AEV/ncgenes7#3]
For binary classification problems, it is good to have a sigmoid postprocessor to map activations to [0 - 1].
This can be part of generic postprocessors.
```
class SigmoidPostProcessor(nc7.model.ModelPostProcessor):
    """
    Sigmoid post-processor to get the probabilities from features
    """
    incoming_keys = ['features']
    generated_keys = ['sigmoid']

    def process(self, features: Union[tf.Tensor, List[tf.Tensor]]
                ) -> Dict[str, tf.Tensor]:
        # pylint: disable=arguments-differ
        # base class has more generic signature
        unit_mapped_activations = tf.nn.sigmoid(features)
        return {"sigmoid": unit_mapped_activations}
```

**Comment by OleksandrVorobiov** [comment]
I guess it is better to have a NonlinearityPostprocessor and then specify which function from tf.nn namespace to use. It will be more generic (possible to use the nucleus7 utils like for activation inside of ModelPlugin).

**Comment by krishna-esrlabs** [comment]
So you propose something like GenericTFLoss right? 
In that case, shall we look for activations in `tf.keras.activations` instead of `tf.nn`?
Because the latter namespace has more functions than we need.

**Comment by OleksandrVorobiov** [comment]
Right
